### DISCUSSION TITLE
Front/Rear cables not traceable anymore in v2.10.4

### DISCUSSION THREAD

**Discussion by Rock-Y** [netbox-community/netbox#5741] [Q&A]
Hi,

I've upgraded our Netbox from 2.9.3 to 2.10.4 last week. Since then cables between front/rear ports are not traceable any more.
Our Datacenter Engineers use this to keep track of all cables between ODF's within our Datacenter.
There are no circuits or interfaces connected as originate or destination on these cables, yet.
Trace with cables connected to an interface or circuit do show the segments.

I think it would be beneficial to be able to trace any cable and view all segments with or without an originate or destination.
If the originate or destination is removed, you still would like to know from which ODF to which ODF cables are connected.

This is probably related to: Improved Cable Trace Performance (#4900)

Examples from v2.9.3:
![Front/Rear cable trace v2.9](https://user-images.githubusercontent.com/852442/106716678-0632f200-65ff-11eb-9896-742fba794a21.PNG)

Example from v2.10.4:
![Front/Rear cable trace v2.10](https://user-images.githubusercontent.com/852442/106716727-1054f080-65ff-11eb-8384-054073c24b6d.PNG)
_Note: Same cable trace after upgrade._

Is this expected behavior for v2.10 (due to #4900) or should I create a bug report for this?

Kind regards,

Bram

**Comment by DanSheps** [comment]
I think cables trace a little different now.

Can you try and replicate this on master.netbox.dev and I can take a look and see what you want and perhaps we can get a FR crafted for eventual inclusion.

**Reply by Rock-Y** [reply]
I have created odf-t1-1, odf-t1-2, odf-t1-3, odf-t1-4 and odf-t1-switch.

Connected rear/front ports 1
master.netbox.dev/dcim/front-ports/75/trace/ -> no segments <-- which we would like to see

Connected rear/front ports 2 with odf-t1-4 rear port 2 to odf-t1-switch ge-0/0/0
master.netbox.dev/dcim/front-ports/76/trace/ -> shows segments

**Reply by DanSheps** [reply]
So looking at this, because there isn't a cable endpoint (an interface) the segments don't get populated into Netbox.

**Reply by Rock-Y** [reply]
An issue for this has been created in #5857 a few days ago.

**Comment by jeremystretch** [comment]
> I've upgraded our Netbox from 2.9.3 to 2.10.4 last week.

Did you run the upgrade script? This calls `manage.py trace_paths` to populate the necessary cable paths.

**Reply by Rock-Y** [reply]
Yes, I've ran the upgrade script it said:
```
[...]
Checking for missing cable paths (python3 netbox/manage.py trace_paths --no-input)...
Found no missing circuit termination paths; skipping
Found no missing console port paths; skipping
Found no missing console server port paths; skipping
Retracing 1 cabled interfaces...
  [####################] 100%
  Retraced 1 interfaces
Found no missing power feed paths; skipping
Found no missing power outlet paths; skipping
Found no missing power port paths; skipping
Finished.
[...]
```
I only have two cable traces in this instance, only one of the two is connected to an interface.

This shows all segments, before:
![netbox 2.9 interface](https://user-images.githubusercontent.com/852442/106957093-da675780-6737-11eb-9737-10cc1541e895.PNG)

After:
![netbox 2.10 interface](https://user-images.githubusercontent.com/852442/106957095-daffee00-6737-11eb-921d-39fca7b1c3a9.PNG)
